src/components/Footer.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+function Footer() {
+    return (
+        <React.Fragment>
+            <div className="footer">
+                <div className style={{ textAlign: 'center' }}>
+                    <div
+                        style={{
+                        height: '40px',
+                        width: '40px',
+                        display: 'inline-block',
+                        margin: '24px',
+                        }}
+                    >
+                        <a href="https://www.facebook.com/groups/codeclannigeria/" target="_blank">
```

Comment:
Codacy found an issue: [Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6058781)